tokenizers/examples/unstable_wasm/README.md

```diff
@@ -0,0 +1,69 @@
+<div align="center">
```

Comment:
We also may want to change the README's content at some point ^^

Reply:
Added a simple line so it's not the pure wasm tutorial (commands explained in it are still useful I find)